lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/ZoomToolbar.java

```diff
@@ -0,0 +1,123 @@
+package nl.tudelft.lifetiles.graph.controller;
+
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+import javafx.beans.value.ChangeListener;
+import javafx.geometry.Orientation;
+import javafx.scene.control.Button;
+import javafx.scene.control.Slider;
+import javafx.scene.control.ToolBar;
+
+/**
+ * Creates a new Toolbar with two buttons and a slider.
+ * These can be used to zoom.
+ *
+ * @author AC Langerak
+ *
+ */
+public class ZoomToolbar {
+
+    /**
+     * The javafx toolbar.
+     */
+    private final ToolBar toolbar;
+
+    /**
+     * Integer which holds the current selected zoomlevel.
+     * Can be listened to.
+     */
+    private final IntegerProperty zoomLevel;
+
+    /**
+     * The maximal amount of zooming.
+     */
+    private final int maxzoom;
+
+    /**
+     * Create a new Toolbar.
+     *
+     * @param zoom
+     *            the maximal value of the zoom
+     */
+    public ZoomToolbar(final int zoom) {
```

Comment:
This class is not an extension of the ToolBar class but rather an implementation of it with added buttons and a custom slider. It does not add new functionalities to ToolBar.